credentials/build.gradle.kts

```diff
@@ -11,31 +11,43 @@ repositories {
   maven("https://repository.jboss.org/nexus/content/repositories/thirdparty-releases/")
 }
 
-val ktor_version = "2.3.4"
-val jackson_version = "2.14.2"
-
 dependencies {
-  api("com.danubetech:verifiable-credentials-java:1.6.0")
-
+  /**
+   * Maintainers - please do not declare versioning here at the module level;
+   * versioning is centralized for the platform in $projectRoot/gradle/libs.versions.toml
+   *
+   * Deps are declared in alphabetical order.
+   */
+
+  // API
+  /*
+   * API Leak: https://github.com/TBD54566975/web5-kt/issues/228
+   *
+   * Change and move to "implementation" when completed
+   */
+  api(libs.comDanubetechVerifiableCredentials)
```

Comment:
Is this is bad because it's a security issue? Or because is it a license-related thing? or both

Reply:
Great question!

API Leaks are not necessarily bad - they could be the intended behaviour. In these cases I've vetted the [inventory](https://github.com/TBD54566975/web5-kt/issues?q=is%3Aissue+is%3Aopen+%22API+Leak%22) with @mistermoe and he indicated to me he'd like to address them.

An API leak means that another project's (let's call that Project X) API is exposed through yours. This has the following implications:

* The consumer of your project needs Project X on its compilation `classpath`
* Project X may bring in a bunch of other dependencies transitively. 
* The more dependencies you require:
  * Higher risk of something in that supply chain having a security vulnerability
  * Higher risk of something in that supply chain using an incompatible license
  * Increased burden on us as project devs to manage the potential version conflicts in a big dependency tree
* You're letting ProjectX's API be part of your project's API surface area. So if they change their API - yours does too, and all consumers must update. You lose authority over locking your API.

in short, API leaks break the software design principle of _encapsulation_, which isn't *necessarily* bad but is something to be inspected on a case by case basis to see if there's a valid reason. In our cases I've inventoried, I'm hearing from the team - nope, let's fix this up and align the Kotlin SDKs with approaches with taken in other language impls.